templates/submission/list.html

```diff
@@ -158,12 +156,11 @@
                 var stats_outdated = false;
 
                 function update_stats() {
-                    return;  // TODO: add back live-updating with graph
-
                     $.ajax({
                         url: '?results'
                     }).done(function (data) {
-                        statistics.html(data);
+                        stats_graph(data);
+                        $('#total-submission-count').text(data.total);
```

Comment:
I know this is here just to prevent adding the count again on the initial load, but I think just having it as part of `stats_graph` makes more sense here.

Reply:
Sure, might as well.

Reply:
Moved it in.